test/pipeline_test_runner.cc

```diff
@@ -202,11 +203,15 @@ vector<ast::ParsedFile> index(unique_ptr<core::GlobalState> &gs, absl::Span<core
         }
 
         unique_ptr<parser::Node> nodes;
-        {
+        if (parser == realmain::options::Parser::SORBET) {
+            std::cout << "Parsing with sorbet" << std::endl;
             core::UnfreezeNameTable nameTableAccess(*gs); // enters original strings
 
             auto settings = parser::Parser::Settings{};
             nodes = parser::Parser::run(*gs, file, settings);
+        } else if (parser == realmain::options::Parser::PRISM) {
+            std::cout << "Parsing with prism" << std::endl;
+            nodes = realmain::pipeline::runPrismParser(*gs, file);
```

Comment:
And here's where we actually use the parser option to decide which parser to use. Sorbet's test implementation is weird, in that it breaks down all the components of the pipeline and tests them separately in order to inject special testing logic between them. Hence, why I had to call `runPrismParser` here. As I've said, I will eventually move that method into its own namespace, which will make it easier to reuse without dipping into pipeline internals.